app/models/batch_process.rb

```diff
@@ -272,6 +289,14 @@ def create_new_parent_csv
   # rubocop:enable  Metrics/CyclomaticComplexity
   # rubocop:enable Metrics/BlockLength
 
+  def valid_viewing_directions
```

Comment:
I think you can replace this method by using the one that's similar to it on the parent object.  https://github.com/yalelibrary/yul-dc-management/blob/main/app/models/parent_object.rb#L73